packages/contracts/test/TestHolderIsReturn.ts

```diff
@@ -0,0 +1,79 @@
+import { loadFixture } from "@nomicfoundation/hardhat-toolbox-viem/network-helpers";
+import { expect } from "chai";
+import hre from "hardhat";
+import { addProject } from "./util";
+
+async function deployMainContract() {
+    return await hre.viem.deployContract("NoneMoney", [], {});
+}
+
+describe("HolderIsReturn", () => {
+    it("Test holder is this project holder settled is return true", async () => {
```

Comment:
對於測試名稱，考慮使用 `Test <Event> (with <Situation>) should <Should>` 來命名，這樣可以很快速瞭解這個測試在做什麼。

```suggestion
    it("Test is holder returned with already returned project should return true", async () => {
```

Reply:
fixed